recipe/meta.yaml

```
@@ -1,31 +1,34 @@
 {% set name = "minio" %}
-{% set version = "7.1.17" %}
+{% set version = "7.2.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: b0b687c1ec9be422a1f8b04c65fb8e43a1c090f9508178db57c434a17341c404
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/minio-{{ version }}.tar.gz
+  sha256: 4b015b018d10c1505f7c3e724fa7c2267760ac7bee6463a624cbf22cd272877b
 
 build:
   noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

```suggestion
  script: {{ PYTHON }} -m pip install . -vv --no-deps
```
I think we set a global environment variable for this.

---

I did try to find our reference for this, but yeah, fewer flags the better until things break.